web/components/answers/answer-resolve-panel.tsx

```diff
@@ -20,6 +20,7 @@ export function AnswerResolvePanel(props: {
     option: 'CHOOSE' | 'CHOOSE_MULTIPLE' | 'CANCEL' | undefined
   ) => void
   chosenAnswers: { [answerId: string]: number }
+  modalSetOpen?: (open: boolean) => void
```

Comment:
This isn't being used as a function in this component. Shouldn't is be a boolean instead? `isInsideModal`?